src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -45,28 +46,6 @@ static VisualStudioInstanceFactory()
         public VisualStudioInstanceFactory()
         {
             AppDomain.CurrentDomain.AssemblyResolve += AssemblyResolveHandler;
-            AppDomain.CurrentDomain.FirstChanceException += FirstChanceExceptionHandler;
-        }
-
-        private static void FirstChanceExceptionHandler(object sender, FirstChanceExceptionEventArgs eventArgs)
```

Comment:
The only downside being that it ties these classes to xUnit instead of allowing them to become part of the core integration framework (and be used with any test framework)